mlflow/genai/optimize/types.py

```diff
@@ -60,10 +60,12 @@ class OptimizerConfig:
         optimizer_llm: Optional LLM parameters for the teacher model. If not provided,
             the target LLM will be used as the teacher.
         algorithm: The optimization algorithm to use. Default: "DSPy/MIPROv2"
+        verbose: Whether to show detailed optimization progress. Default: False
```

Comment:
```suggestion
        verbose: Whether to show stdout and stderr during optimization. Default: False
```

Reply:
This is accurate for MIPRO, but might be too specific to the current implementation. Different optimizers might have different log strategy during optimization.